spec/integration/transport/winrm_spec.rb

```diff
@@ -47,7 +50,14 @@ def mk_config(conf)
 PS
 
   def make_target(host_: host, port_: port, conf: config)
-    Bolt::Target.new("#{host_}:#{port_}").update_conf(conf.transport_conf)
+    t = inventory.get_target("#{host_}:#{port_}")
+    update_target(t, conf.transports[conf.transport.to_sym])
+    t
+  end
+
+  def update_target(targ, conf)
+    transport_config = targ.options.merge(conf)
+    targ.inventory_target.set_config(targ.transport, transport_config)
   end
```

Comment:
I'd like to look at using `Bolt::Target#from_hash` here instead... creating a config object to create a target, and using `set_config` to modify settings seems unnecessary, since all we care about is having a single target with the appropriate config set.